vassal-app/src/main/java/VASSAL/counters/BoundsTracker.java

```diff
@@ -43,6 +43,14 @@ public void clear() {
 
   public void addPiece(GamePiece p) {
     if (p.getMap() != null) {
+      if(p.getParent() != null) {
```

Comment:
Checkstyle is complaining about the lack of a space after the `if`.

Reply:
Yeah, sorry. I should be better about doing a maven build before committing. New habits.